app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Posts of the user</h1>
\ No newline at end of file
+<%= render partial: "users/render", users: @users%>
+
+<% @posts.each do |post| %>
+<ul class="main-container">
+ <li class="blog">
+      <a href="<%= user_post_path(post.author_id, post.id) %>" class="blog">
+        <h2><%= post.title %></h2>
+        <p><%= post.text %></p>
+        <div class="features">
+        <span>Comments: <%= post.comments.length %></span>
+        <span>Likes: <%= post.likes.length %></span>
+        </div>
+      </a>
```

Comment:
- [ ] [Optional] Same here, It would be better professionally if you used [link_to](https://www.rubyguides.com/2019/05/rails-link_to-method/) method instead of using the regular anchor tag `<a>`.